docs/layouts/shortcodes/build_table.html

```diff
@@ -0,0 +1,35 @@
+<style type="text/css">
+  .build-table  {border-collapse:collapse;border-spacing:0;}
+  .build-table th{font-weight: bold;}
+  .build-table th span.reduce-impact{font-weight: normal;}
+  .build-table span.code{color:#905;background-color:#DDD}
+  </style>
+  <table class="build-table">
+    <thead>
+      <tr>
+        <th colspan="2">magic-modules - build variables</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th>OUTPUT_PATH <span class="reduce-impact">(required)</span> </th>
+        <td >The location of the provider you are unioning generation into.</td>
+      </tr>
+      <tr>
+        <th>VERSION <span class="reduce-impact">(required)</span></th>
+        <td>The version of the provider you are building into. <br/> Valid values are <span class="code">'ga'</span> and <span class="code">'beta'</span>.</td>
+      </tr>
+      <tr>
+        <th >ENGINE <span class="reduce-impact">(optional)</span></td>
```

Comment:
There are some extraneous spaces in the opening HTML tags left over from when these had classes

```suggestion
        <th>ENGINE <span class="reduce-impact">(optional)</span></td>
```